### ISSUE TITLE
Development of RAM Engagement Prioritisation Template

### ISSUE THREAD

**Issue by f-rower** [alan-turing-institute/research-application-management#67]
### Provide a description

Links to issue https://github.com/alan-turing-institute/research-application-management/issues/66

**Comment by f-rower** [comment]
- RAG rating
- Prioritisation Dimensions
- Add graphic that shows project lifecycle

**Comment by f-rower** [comment]
This is the current set of proposed dimensions for RAM *project* engagement prioritisation (tested in DCE context):
| Dimension | Description |
|------------|------------|
| Alignment with Turing Grand Challenges | Does the project align well with Turing Grand Challenges? (Environment and Sustainability, Healthcare, Defence)|
| Alignment with programme objectives and priorities| Does the project align well with the programme objectives, priorities and themes?|
| Project Status | Is the project in an exploratory phase, ongoing, finalised and in need of further impact exploration? |
| Opportunities for RAM engagement | What is there for a RAM to do in the project? The opportunities for engagement can be further examined in terms of RAM expertise, interest, whether the activities can be conducted by others... |
| Readiness for RAM engagement | Can the RAM be involved straight away? in the near future? far future? |
| Interest in RAM engagement | How much interest is there in the project team for a RAM to be engaged? Has the RAM role been mentioned? Is there resistance towards RAM engagement? Has RAM engagement been considered? |
| Programme leadership buy-in | Does programme leadership endorse RAM engagement in the project? |

**Comment by harisood** [comment]
This looks great @f-rower ! My understanding is that then this issue is basically going into detail on step 3 - we should probs create separate ones for step 1, step 2, and the overall plan (which tbh I think can be #66 )

**Comment by f-rower** [comment]
I agree with the sentiment, but before we make too many changes, perhaps let's bring up this plan at next Monday's meeting @harisood so the team is somewhat aligned on what's happening?

**Comment by harisood** [comment]
@f-rower is this what you had in mind for bullet 3 above?

| Dimension | Description | R | A | G |
|------------|------------| - | - | - |
| Alignment with Turing Grand Challenges | Does the project align well with Turing Grand Challenges? (Environment and Sustainability, Healthcare, Defence)| No alignment to any grand challenge | Can fit under a grand challenge but link is unclear/tenuous | Project clearly fits within one or more of Turing GCs
| Alignment with programme objectives and priorities| Does the project align well with the programme objectives, priorities and themes?| No alignment with defined programme priorities/themes | Project aligns loosely with programme priorities/themes, or aligns with some and not others | Project clearly and directly aligned with programme objectives, priorities and themes
| Project Status | Is the project in an exploratory phase, ongoing, finalised and in need of further impact exploration? | N/A - should not be part of this matrix (this is factual and not RAG-able) | N/A - should not be part of this matrix (this is factual and not RAG-able) | N/A - should not be part of this matrix (this is factual and not RAG-able)
| Opportunities for RAM engagement | What is there for a RAM to do in the project? The opportunities for engagement can be further examined in terms of RAM expertise, interest, whether the activities can be conducted by others... | No opportunity for RAM engagment, or expectation for RAM engagement is misaligned with RAM team skillset/values (e.g. just wanting administrative support) | RAM opportunities may exist but are unclear, or limited buy in from the team (this is the kind of sitch where a RAM may be best) | RAM opportunity is extremely clear and well-defined (e.g. to run a user testing cycle) (this is the kind of sitch where a RAO may be best)
| Readiness for RAM engagement | Can the RAM be involved straight away? in the near future? far future? | Unclear how RAM would fit into team | Ability to onboard RAM is there, but loosely defined, or RAM may have to do a lot of people management to prepare the team for RAM work | Team is ready for RAM to be immediately onboarded and become central part of team
| Interest in RAM engagement | How much interest is there in the project team for a RAM to be engaged? Has the RAM role been mentioned? Is there resistance towards RAM engagement? Has RAM engagement been considered? | No interest/no understanding of what RAM is | Team is aware of RAMs, but maybe not proactively seeking RAM engagement. Encouragement is required to be fully sold on RAMs | Team is aware of RAMs and how they can help their team | 
| Programme leadership buy-in | Does programme leadership endorse RAM engagement in the project? |  See above, update context for programme leadership | See above, update context for programme leadership | See above, update context for programme leadership |

**Comment by harisood** [comment]
It's pretty basic but some initial thoughts there!

**Comment by f-rower** [comment]
@harisood is cool and I have worked on an iteration of the RAM Matrix and template for RAM project engagement prioritisation, which is currently hosted [here[(https://thealanturininstitute-my.sharepoint.com/:w:/g/personal/fgomezmedina_turing_ac_uk/EQDLdl0vzxFGgQHLCtFqqjgB3Dmj86OgeUhJmlz9ve8Fmw?e=sk9aDY)

**Comment by f-rower** [comment]
Thoughts following from discussion during RAM meeting 11/3/2024
- Adding RAM prioritisation framework to RAM chapter in TPS
- Adapting RAM prioritisation framework as a tool for informing translation and pull-through to adoption by Innovation directors
- Need to decide what specifically we want this matrix for
